### PR TITLE
Expose internal fn to load ops for a toplevel

### PR DESCRIPTION
**Pull request by StachuDotNet**
The results are currently aren't ideal, but it's better than nothing.

needs tests

### CODE REVIEW

**Comment by StachuDotNet** [comment]
Hm, Actually I don't think I can cover this in a unit test with the existing infra. (at least, not in a .test file) since we don't have fns to create/manipulate handlers/etc. WDYT, @pbiggar?

File: fsharp-backend/src/BackendOnlyStdLib/LibDarkInternal.fs [hunk 1]
```diff
@@ -1170,4 +1170,27 @@ human-readable data."
...
+          | _, [ DUuid canvasID; DInt tlid ] ->
+            uply {
+              let tlid = uint64 tlid
+              let! ops =
+                let loadAmount = Serialize.LoadAmount.IncludeDeletedToplevels
+                Serialize.loadOplists loadAmount canvasID [ tlid ]
+
+              match ops with
+              | [ (_tlid, ops) ] -> return ops |> List.map (string >> DStr) |> DList
+              | _ -> return DList []
```
Comment (StachuDotNet) [inline fsharp-backend/src/BackendOnlyStdLib/LibDarkInternal.fs:1191]: `Serialize.loadOplists` does not return a `(tlid, ops)` pair for TLIDs that either don't exist or have no ops.

This (returning a `DList []` in such a case) seemed like a reasonable response to such, but maybe not?

Alternatively, we could update `Serialize.loadOplists` to return a pair regardless of whether it has ops or not.

Comment (StachuDotNet) [reply fsharp-backend/src/BackendOnlyStdLib/LibDarkInternal.fs:1191]: draft refactor of `Serialize.loadOplists` (just the end)

```
  Sql.query query
  |> Sql.parameters [ "canvasID", Sql.uuid canvasID; "tlids", Sql.idArray tlids ]
  |> Sql.executeAsync (fun read -> (read.tlid "tlid", read.bytea "oplist"))
  |> Task.bind (fun list ->
    tlids
    |> Task.mapWithConcurrency 2 (fun tlid ->
      match List.tryFind (fun (t, _ops) -> tlid = t) list with
      | Some (_tlid, oplist) ->
        task { return (tlid, BinarySerialization.deserializeOplist tlid oplist) }
      | None -> task { return (tlid, []) }))
```

Comment (pbiggar) [reply fsharp-backend/src/BackendOnlyStdLib/LibDarkInternal.fs:1191]: I would make the change in darkinternal. Empty list seems fine.

**Comment by StachuDotNet** [comment]
currently a minor build error, which I'll fix in a bit. Short of that, seems good to me.

**Review by pbiggar — APPROVED** [review]

**Pull request merged**